file_writer_control/CommandStatus.py

```diff
@@ -0,0 +1,96 @@
+from enum import Enum, auto
+from datetime import datetime
+
+
+class CommandState(Enum):
+    """
+    The state of a command.
+    """
+    UNKNOWN = auto()
+    NO_COMMAND = auto()
+    WAITING_RESPONSE = auto()
+    TIMEOUT_RESPONSE = auto()
+    ERROR = auto()
+    SUCCESS = auto()
+
+
+class CommandStatus(object):
+    """
+    The status of a command.
+    """
+    def __init__(self, job_id: str, command_id: str):
+        self._job_id = job_id
+        self._command_id = command_id
+        self._last_update = datetime.now()
+        self._state = CommandState.NO_COMMAND
+        self._message = ""
+
+    def __eq__(self, other):
```

Comment:
May be we can give more meaningful names to the args.

Reply:
Also may be provide the type for the arg `other`?

Reply:
I missed the type here, it has now been added. Feel free to suggest a better name for the argument.

Reply:
I would have used `other_status`, as it is basically a status and we are checking if it is an instance of `CommandStatus`. But `other` is fine, if you prefer it.

Reply:
I think that `other_status` is redundant, especially after having added the type to the function declaration. I have changed it though.